api/TimeAddressableMediaStore.yaml

```diff
@@ -512,16 +512,16 @@ paths:
         "http_object_store" media store type which will include a "get_url" property that contains a
         HTTP URL for downloading the media object.
 
-        The flow segment may include timing and range adjustment information that the client needs to
+        The flow segment may include timing and timerange adjustment information that the client needs to
         apply when extracting the samples from the media object.
-        - The range of samples must be limited to the range from `sample_offset` to `sample_offset + sample_count`
-          (exclusive) if `sample_count` is set, or from `sample_offset` until the end if `sample_count`
-          is not set.
+        - The timerange of samples must be limited to `sample_offset` to
```

Comment:
This is a bit of a mega-sentence. How about:
```
- If `sample_count` is set, the timerange of samples must be `sample_offset` to `sample_offset + sample_count`
- Otherwise, the timerange of samples must be `sample_offset` until the end.
```